### DISCUSSION TITLE
cmus maintenance

### DISCUSSION THREAD

**Discussion by flyingmutant** [cmus/cmus#1048] [General]
Continuing from https://github.com/cmus/cmus/issues/856!

**Comment by pgaskin** [comment]
Should we close #856 now that it's a bit more active (I'll keep responding to bugs and maybe implement features as they come up) and the information there's a bit outdated?

**Reply by flyingmutant** [reply]
Hm, maybe? Despite me doing nothing, cmus does chug along a bit indeed. @nefthy do you think there is a reason to keep it (and note on top of the README)?

**Reply by pgaskin** [reply]
Yes, and it seems like @gavtroy is planning to do some work on cmus too.

**Reply by nefthy** [reply]
I think we can ditch it, if you are happy continuing to do the merging/release.

**Reply by flyingmutant** [reply]
Great, let's do it then. @pgaskin can you please make a PR closing #856 and removing the README note?

**Reply by DusanLesan** [reply]
@flyingmutant @nefthy Why was `not actively maintained` README note not removed? This repo is active enough and the note may deter potential users and future maintainers. As long as Gregory is willing to merge fixes, project is alive in my opinion.

**Reply by nefthy** [reply]
Oversight corrected in #1129

**Reply by pgaskin** [reply]
I must have missed this notification earlier...

**Comment by pgaskin** [comment]
Also, I personally think we should remove the other discussion categories for now so everything goes into General.

**Reply by flyingmutant** [reply]
Good idea! I've left Q&A as well for now, maybe it fill find some use.

**Reply by pgaskin** [reply]
I actually think we should remove that one too, at least until we figure out what people want to ask.

**Reply by flyingmutant** [reply]
Done.

**Comment by pgaskin** [comment]
v2.9.1 and the MPRIS dependency fix are in the official Debian/Ubuntu packages now.
